odoo/addons/base/models/res_users_deletion.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class ResUsersDeletion(models.Model):
+    """User deletion requests.
+
+    Those requests are logged in a different model to keep a trace of this action and the
+    deletion is done in a CRON. Indeed, removing a user can be a heavy operation on
+    large database (because of create_uid, write_uid on each model, which are not always
+    indexed).
+    """
+    _name = 'res.users.deletion'
+    _description = 'Users Deletion Request'
+
+    # Integer field because the related user might be deleted from the database
+    user_id = fields.Integer('User Id', required=True)
+
+    state = fields.Selection([('todo', 'To Do'), ('done', 'Done'), ('fail', 'Failed')], required=True, string='State')
+    user_create_date = fields.Datetime('User Create Date', help='When the related user has been created')
```

Comment:
Why do we keep that exactly? Information / statistics?

Reply:
it was suggested by odo, but not really useful, I remove it